prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Brandon Au';
+    console.log(fullName);
+    var numberOfPets = 0;
+    console.log(numberOfPets);
+    var likesVegetables = true;
+    console.log(likesVegetables);
+    var nothing = null;
+    console.log(nothing);
+    var never = '';
+    console.log(never);
```

Comment:
This `never` variable actually contains an "empty" string, but if you want to leave it uninitialized, just a semicolon will do.

```js
var never;
```

Reply:
![prep-javascript-variables-screenshot-updated](https://user-images.githubusercontent.com/109652009/183560299-aef7e513-2272-45bd-9d26-ccc887f7d3c0.PNG)